plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -296,7 +296,7 @@ class CodeWhispererCodeScanManager(val project: Project) {
                 if (e.cause?.message?.contains("com.intellij.openapi.compiler.CompilerPaths") == true) {
                     message("codewhisperer.codescan.java_module_not_found")
                 } else {
-                    message("codewhisperer.codescan.service_error")
```

Comment:
Why is this error removed? what case is this message handling here?

Reply:
Here modified error should be `message("codewhisperer.codescan.run_scan_error")` which is by default error so I removed this error
To be precise we can remove even this null.

Reply:
so we're catching the null error later?

Reply:
No if we keep it `null` then the `errorMessage`  is auto assigned to the default value that is `message("codewhisperer.codescan.run_scan_error")`.